sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -172,100 +172,99 @@ def deserialization_callback(
         )
 
     @distributed_trace
-    def get_job_status(self, job_id, **kwargs):
-        # type: (str, **Any) -> JobStatusResult
-        """Gets the status of a translation job.
+    def get_translation_status(self, translation_id, **kwargs):
```

Comment:
If given a translation ID (especially in the situation of rehydration / I'm not the one who started the operation), I could see using this method to check the status of the translation first to see if I even need to rehydrate. It's pretty similar to `get_custom_model` in FR, so I see no reason not to expose it. 

Thinking on the naming concern you brought up. I think we probably want to be consistent with what we have for the document methods. I think it's important to keep the "status" part in the name since you're not getting back the actual resource. Which means I guess we'd go with `list_all_translation_statuses`. Thoughts?

Reply:
I think for now we can have it as `list_all_translation_statuses` and after this preview, we can sit down and look at all API and see if we want to keep `statuses` or go with wat LLC is doing

Reply:
Sounds good. Will .NET make the change too? I thought you guys had `GetTranslations`

Reply:
I was thinking not for this release and wait for the conversation next sprint. If we decided to leave the `statuses` I will add it then

Reply:
I will rename it too for consistency